Client/Configuration/Fennec.xcconfig

```diff
@@ -12,4 +12,9 @@ INCLUDE_SETTINGS_BUNDLE = YES
 
 // Adjust SDK Variables
 ADJUST_ENVIRONMENT =
-ADJUST_APP_TOKEN = 
+ADJUST_APP_TOKEN =
+
+// Leanplum SDK Variables
+LEANPLUM_ENVIRONMENT = development
+LEANPLUM_APP_ID = 
+LEANPLUM_KEY =
```

Comment:
@Sdaswani Just keep them around in your local checkout. We can also investigate if we can include them in like a `Local.xcconfig` that is ignored by Git. THen you can just keep that file around for local Fennec builds and have all the settings in there. (Also useful for other things like Sentry and Adjust)